hackathon_site/review/tests.py

```diff
@@ -1,3 +1,267 @@
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertNotContains(response, "Send Decisions")
+
+    def test_sends_quantity_number_of_accepted_emails(self):
+        self._login()
+        self._create_teams_and_reviews_for_mail_tests()
+
+        quantity_before = Review.objects.filter(
+            decision_sent_date__isnull=True, status="Accepted"
+        ).count()
+
+        self.form_data["quantity"] = 3  # Send 3 acceptance emails
+
+        response = self.client.post(self.view, data=self.form_data)
+
+        quantity_after = Review.objects.filter(
+            decision_sent_date__isnull=True, status="Accepted"
+        ).count()
+
+        self.assertEqual(len(mail.outbox), 3)
+        self.assertRedirects(response, self.view)
+        self.assertEqual(quantity_before, quantity_after + self.form_data["quantity"])
+
+    def test_send_all_accepted_within_current_date_range(self):
+        self._login()
+        self._create_teams_and_reviews_for_mail_tests(date_offset=5)
+
+        quantity_before = Review.objects.filter(
+            decision_sent_date__isnull=True, status="Accepted"
+        ).count()
+
+        # Send 10 acceptance emails. There's only 8 accepted people, and 1 older. So should
+        # only send 7 emails
+        self.form_data["quantity"] = 10
+        self.form_data["status"] = "Accepted"
+
+        response = self.client.post(self.view, data=self.form_data)
+
+        quantity_after = Review.objects.filter(
+            decision_sent_date__isnull=True, status="Accepted"
+        ).count()
+
+        self.assertEqual(len(mail.outbox), 7)
+        self.assertRedirects(response, self.view)
+        self.assertEqual(quantity_before, quantity_after + 7)
+
+    def test_correct_text_in_accepted_email(self):
+        self._login()
+        self._create_teams_and_reviews_for_mail_tests()
+
+        # Send 1 accepted email
+        response = self.client.post(self.view, data=self.form_data)
+
+        clean = re.compile("<.*?>")
+        clean_mail_body = re.sub(clean, "", mail.outbox[0].body)
+
+        link = f"http://testserver{reverse('event:dashboard')}"
+
+        rsvp_deadline = (
+            datetime.now().date() + timedelta(days=settings.RSVP_DAYS)
+        ).strftime("%b %d %Y")
+
+        self.assertIn(link, mail.outbox[0].body)
+        self.assertIn(rsvp_deadline, mail.outbox[0].body)
+        self.assertIn(settings.HACKATHON_NAME, mail.outbox[0].body)
+        self.assertIn(settings.PARTICIPANT_PACKAGE_LINK, mail.outbox[0].body)
+        self.assertIn(settings.CHAT_ROOM_LINK, mail.outbox[0].body)
+        self.assertIn(
+            f"Congratulations, you’ve been accepted to { settings.HACKATHON_NAME }",
+            mail.outbox[0].subject,
+        )
+        self.assertIn(
+            f"The {settings.HACKATHON_NAME} team has reviewed your application, and we’re excited to welcome you to CoolHacks!",
```

Comment:
`CoolHacks` hard coded at the end of the sentence, can cut that out or replace it with another `{settings.HACKATHON_NAME}`